auth-tokens/src/main/java/com/palantir/tokens/auth/ImmutablesStyle.java

```diff
@@ -23,6 +23,6 @@
 import org.immutables.value.Value;
 
 @Target({ElementType.PACKAGE, ElementType.TYPE})
-@Retention(RetentionPolicy.CLASS)
+@Retention(RetentionPolicy.SOURCE)
 @Value.Style(visibility = Value.Style.ImplementationVisibility.PACKAGE, jdkOnly = true)
-public @interface ImmutablesStyle {}
+@interface ImmutablesStyle {}
```

Comment:
Unfortunately, this annotation is used in a handful of internal repos. Is this break acceptable?

Excavating out the change is a bit cumbersome because it may involve adding a dependency on our internal `immutables-style` package.

Reply:
so why change the visibility now? how about just fix the compile warning?

Reply:
Sure I guess I could open a second PR. I'd like to make both changes though.

Reply:
eh, if we're eventually making both no point in 2 prs